escrow/scripts/deploy.js

```diff
@@ -0,0 +1,47 @@
+const { Contract } = require("ethers");
+const { calcEthereumTransactionParams } = require("@acala-network/eth-providers");
+const { ACA } = require("@acala-network/contracts/utils/Address");
+
+const TokenContract = require("@acala-network/contracts/build/contracts/Token.json");
+
+const txFeePerGas = '199999946752';
+const storageByteDeposit = '100000000000000';
+
+const ESCROW_AMOUNT = 1_000_000_000; // 0.001 ACA
```

Comment:
You could use `formatUnits` within the script or maybe even here in stead of manually typing the `0`'s